osquery/tables/networking/linux/iptables.cpp

```diff
@@ -71,6 +72,39 @@ void parseIpEntry(const ipt_ip *ip, Row &r) {
   r["outiface_mask"] = TEXT(outiface_mask);
 }
 
+void parseEntryMatch(const struct ipt_entry* en, Row& r) {
+  // Get rule port details from the xt_entry_match object
+
+  auto m = (struct xt_entry_match*)en->elems;
```

Comment:
Can you also check that `m != nullptr` after this?
```cpp
if (m == nullptr) {
  return;
}
```

Reply:
@theopolis I added this previously, but I was getting a compiler warning that the expression will always be true. So I removed it.